vite.config.js

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  define: {
+    global: {},
+    "process.env": process.env,
```

Comment:
You might be able to get away without this. I was trying to retain the format of constants.js but it wouldn't work.

Reply:
yeah seems to work without it